### PR TITLE
chore: run browser tests on both Chromium and Webkit

### LINKED ISSUE #40: Browser tests fail on Chromium
### Aim

I'm trying to test the simulator in all main browser environments.

### Expected Behavior

The browser tests should pass on all 3 of Chromium, Firefox and Webkit browsers

### Bug

Browser tests fail on Chromium

### To Reproduce

1.
2.
3.
4.

### Environment

_No response_

### Additional Context

_No response_

### Would you like to submit a PR for this Issue?

No

### Support Needs

_No response_

### PR DESCRIPTION
**Pull request by TomAFrench**
# Description

## Problem\*

Resolves #40 

## Summary\*

This PR sets out to ensure that the browser tests run on all relevant browsers.

## Additional Context

# PR Checklist\*

- [ ] I have tested the changes locally.
- [ ] I have formatted the changes with [Prettier](https://prettier.io/) and/or `cargo fmt` on default settings.

### CODE REVIEW

File: src/barretenberg/mod.rs [hunk 1]
```diff
@@ -36,6 +35,8 @@ pub(crate) enum FeatureError {
     InvalidUsize { value: i32, source: std::num::TryFromIntError },
     #[error("Value expected to be 0 or 1 representing a boolean")]
     InvalidBool,
+    #[error("Black Box vendor Instance creation failed")]
+    BlackBoxInstanceFailed,
```
Comment (TomAFrench) [inline src/barretenberg/mod.rs:39]: Does this given us any benefit? We immediately unwrap this error anyway.

File: src/barretenberg/pedersen.rs [hunk 2]
```diff
@@ -24,21 +24,21 @@ impl Pedersen for Barretenberg {
...
-        "0x0ecf9d98be4597a88c46a7e0fa8836b57a7dcb41ee30f8d8787b11cc259c83fa",
-    )
-    .unwrap();
-
-    assert_eq!(expected_x.to_hex(), x.to_hex());
-    assert_eq!(expected_y.to_hex(), y.to_hex());
-    Ok(())
-}
+// #[test]
+// fn pedersen_hash_to_point() -> Result<(), Error> {
```
Comment (TomAFrench) [inline src/barretenberg/pedersen.rs:28]: Let's restore these tests.

Comment (kobyhallx) [reply src/barretenberg/pedersen.rs:28]: Would we not want to have these tests now in mocha?

File: src/barretenberg/mod.rs (hunk 1 above)
Comment (kobyhallx) [reply src/barretenberg/mod.rs:39]: Yes, we can get rid of this, I was going somewhere else with it...

**Pull request merged**